src/controllers/notificationController.ts

```diff
@@ -71,7 +71,7 @@ export default class NotificationController {
   > {
     return withUserFromRequest(req, async user =>
       withValidatedOrValidationError(
-        InstallationID.decode(req.params.id),
+        InstallationID.decode(req.params.id), // An installation could be overwritten from any client passing an existing installationID
```

Comment:
this commet is somewhat obscure (it's a bug or feature ? in which case it happens ?)

Reply:
This was an old memo to me, it was removed in fd811c7